applications/nrf_desktop/bluetooth.rst

```diff
@@ -201,6 +201,12 @@ After a successful erase advertising procedure, the peripheral removes all of th
 
 Apart from that, the following changes are applied in configurations that support Fast Pair:
 
+* The :kconfig:option:`SB_CONFIG_BT_FAST_PAIR` Kconfig option is enabled in the sysbuild configuration.
+  With this option enabled:
```

Comment:
Consider referring to doc added by @kapi-no here: https://github.com/nrfconnect/sdk-nrf/pull/16022 (to avoid duplicating information from there)

Reply:
If mentioned PR will be merged first then I'll address it.

Reply:
The mentioned PR is already approved, so should go in quickly

Reply:
Link here would be beneficial for the reader I think

Reply:
will link to the ``` ref:`ug_bt_fast_pair_prerequisite_ops_kconfig` ``` once https://github.com/nrfconnect/sdk-nrf/pull/16022 is merged

Reply:
@MarekPieta so instead the description what this option does we just want something like:
```suggestion
  See ref:`ug_bt_fast_pair_prerequisite_ops_kconfig` for the details.
```
?

Reply:
Correct

Reply:
applied